client/components/Header.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/extensions */
+import React from 'react';
+import styled from 'styled-components';
+import NavButtons from './NavButtons.jsx';
+
+const Heading = styled.h2`
+  background-color: inherit;
+  display: inherit;
```

Comment:
give app.jsx all the default values so the child nodes like heading and section will inherit the styling by default